drivers/clock_control/clock_stm32f2_f4_f7.c

```diff
@@ -56,6 +56,9 @@ uint32_t get_pllsrc_frequency(void)
 __unused
 void config_pll_sysclock(void)
 {
+#if defined(STM32_SRC_PLL_R) & STM32_PLL_R_ENABLED & defined(RCC_PLLCFGR_PLLR)
```

Comment:
Are you sure it shouldn't be `&&` instead?

Reply:
Done. [Copy-paste](https://github.com/zephyrproject-rtos/zephyr/blob/main/drivers/clock_control/clock_stm32_ll_common.c#L517) error